registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/WorkflowInternalActionCode.java

```diff
@@ -1,6 +1,7 @@
 package io.mosip.registration.processor.core.code;
 
 
+// TODO: Auto-generated Javadoc

```

Comment:
Why is this TODO added ?

Reply:
while adding java doc for new code its added ,fixed